### ISSUE TITLE
AE2 integration cc-tweaked-1.12.2-1.84.0.jar

### ISSUE THREAD

**Issue by SYMemy** [SquidDev-CC/CC-Tweaked#382]
I am trying to write a program for my AE2 System, but I can't find a way to get an AE2 API. 
I found this thread on the computercraft forum [computercraft forum](http://www.computercraft.info/forums2/index.php?/topic/22537-1710-cc-and-applied-energistics-2/), someone suggested a pastebin code to display all methods of a peripheral next to the computer. Whenever I try to run his code, it says that there is no peripheral on that side. 

To my Question, is there a way to interact with AE2, or do I need to install some other mods?
And I am sorry to post this here, but I didn't know where to ask.

**Comment by Wojbie** [comment]
You do need some kind of mod that adds CC integration to AE2 blocks. I would suggest using Plethora (also made by SquidDev) as it provides integration for most commonly used mods.

**Comment by SYMemy** [comment]
Thank you very much!
Is there a Discord or something If I got more questions?

**Comment by lupus590** [comment]
https://discord.gg/H2UyJXe

**Comment by kid2407** [comment]
Perhaps I did not grasp the work of plethora correctly, but here we go:

What I want: Access the data an ME System has, e.g. items stored, stored power, crafting cpus/jobs etc.

From what I have gathered in the plethora docs: This is not included in what the mod provides, just a basic-ish "this is a ae2 block" data.

If not covered elsewhere, where would I need to hook in with an own sort of "cc extras for ae2 mod" to add methods a cc computer can see / use?
